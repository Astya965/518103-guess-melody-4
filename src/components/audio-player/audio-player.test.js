import React from "react";
import renderer from "react-test-renderer";

import AudioPlayer from "./audio-player.jsx";

it(`Render AudioPlayer`, () => {
  const tree = renderer
    .create(
        <AudioPlayer
          isStarting={false}
          src={`some.mp3`}
          onPlayButtonClick={() => {}}/>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
