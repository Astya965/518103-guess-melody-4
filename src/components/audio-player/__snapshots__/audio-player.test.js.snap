// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render AudioPlayer 1`] = `
Array [
  <button
    aria-label="Play button"
    className="track__button track__button--play"
    disabled={true}
    onClick={[Function]}
    type="button"
  />,
  <div
    className="track__status"
  >
    <audio
      src="some.mp3"
    />
  </div>,
]
`;
